source/install/docker-upgrading.rst

```diff
@@ -1,146 +1,170 @@
 the update procedure does not complete successfully or if you decide not to
-upgrade your |pmm-server| after trying the new version.
+upgrade your PMM Server after trying the new version.
 
-The |docker.stop| command stops the currently running |opt.pmm-server| container:
+The ``docker stop`` command stops the currently running ``pmm-server`` container:
+
+.. code-block:: bash
 
-.. include:: ../.res/code/docker.stop.pmm-server.txt
+   $ docker stop pmm-server
 
-The following command simply renames the current |opt.pmm-server| container to
+The following command simply renames the current ``pmm-server`` container to
 avoid name conflicts during the update procedure:
 
-.. include:: ../.res/code/docker.rename.pmm-server.pmm-server-backup.txt
+.. code-block:: bash
 
-.. _pmm.deploying.docker-image.pulling:
+   $ docker rename pmm-server pmm-server-backup
 
-`Pulling a new Docker Image <docker-upgrading.html#image-pulling>`_
---------------------------------------------------------------------------------
+.. _pmm.deploying.docker-image.pulling:
+.. _image-pulling:
 
-|docker| images for all versions of |pmm| are available from
-`percona/pmm-server`_
-|docker| repository.
+**************************
+Pulling a new Docker Image
+**************************
 
-When pulling a newer |docker| image, you may either use a specific version
-number or the ``2`` image which always matches the highest version
+When pulling a newer image, you may either use a specific version
+number, or the ``2`` image which always matches the highest version
 number. 
 
-This example shows how to pull a specific version:
+**Examples**
+
+To pull a specific version:
 
 .. code-block:: bash
 
-   $ docker pull percona/pmm-server:2.8.0
+   $ docker pull percona/pmm-server:2.9.0
+
+To pull the latest PMM 2 version:
+
+.. code-block:: bash
+
+   $ docker pull percona/pmm-server:2
+
 
-This example shows how to pull the latest PMM 2 version:
-   
-.. include:: ../.res/code/docker.pull.percona-pmm-server-latest.txt
-   
 .. _pmm.deploying.docker-container.creating:
+.. _container-creating:
+
+******************************************************
+Creating a new Docker container based on the new image
+******************************************************
 
-`Creating a new Docker container based on the new image <docker-upgrading.html#container-creating>`_
--------------------------------------------------------------------------------------------------------
+After you have pulled a new version of PMM from the Docker repository, you can
+use ``docker run`` to create a ``pmm-server`` container using the new image.
 
-After you have pulled a new version of |pmm| from the |docker| repository, you can
-use |docker.run| to create a |opt.pmm-server| container using the new image.
+.. code-block:: bash
 
-.. include:: ../.res/code/docker.run.latest.txt
+   $ docker run -d -p 80:80 -p 443:443 --volumes-from pmm-data \
+      --name pmm-server --restart always percona/pmm-server:2
```

Comment:
I believe with the percona/pmm-server:2 command it's actually doing the docker pull but recognizing that all components have been downloaded already...you can actually skip the docker pull command above if you're putting it this way.